editor/app/src/main/res/layout/quiz_panel_fragment.xml

```diff
@@ -1,13 +1,83 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
Check out the Design mode to see how it looks. Currently, your views are stacked on top of one another.